rec/src/NA6PFastTrackFitter.cxx

```diff
@@ -247,6 +247,15 @@ bool NA6PFastTrackFitter::updateTrack(NA6PTrack* trc, NA6PBaseCluster* cl) const
   return true;
 }
 
+// Explicit instantiations to ensure symbols are emitted in the library
+template void NA6PFastTrackFitter::addCluster<NA6PMuonSpecCluster>(int, NA6PMuonSpecCluster*);
+template void NA6PFastTrackFitter::addCluster<NA6PVerTelCluster>(int, NA6PVerTelCluster*);
+template void NA6PFastTrackFitter::addCluster<NA6PBaseCluster>(int, NA6PBaseCluster*);
```

Comment:
OK, this frees from the necessity to have the templated method definition in the header file, still, not clear why we need 3 methods, if the underlying pointer is anyway  `NA6PBaseCluster*`?